src/GraphBuilder.js

```diff
@@ -28,13 +28,15 @@ export default class GraphBuilder {
   }
 
   buildSubject(subject, subjectTerm) {
-    this.dataset.add(
-      rdf.quad(
-        subjectTerm,
-        rdf.namedNode("http://www.w3.org/1999/02/22-rdf-syntax-ns#type"),
-        rdf.namedNode(subject.subjectTemplate.class)
+    subject.classes.forEach((clazz) => {
```

Comment:
Add the classes from the subject (i.e., the classes selected by the user), not the subject template.

Reply:
If this needs a comment - maybe put it in the code?